services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentContentService.kt

```diff
@@ -48,9 +53,14 @@ class ApplicationAttachmentContentService(
                 throw AttachmentNotFoundException(attachmentId)
             }
 
+    fun exists(location: String): Boolean = fileClient.exists(Container.HAKEMUS_LIITTEET, location)
+
+    fun existsForApplication(applicationId: Long): Boolean =
+        fileClient.existsByPrefix(Container.HAKEMUS_LIITTEET, prefix(applicationId))
```

Comment:
These are not called from outside tests. Tests can use the FileClient directly.

Reply:
Removed "exists" methods